src/route.rs

```diff
@@ -67,19 +70,19 @@ impl<'a> TryFrom<seed::Url> for Route<'a> {
             Some("settings") => Some(Route::Settings),
             Some("profile") => path
                 .next()
-                .filter(non_empty)
+                .filter(|v| !v.is_empty())
```

Comment:
Why have you changed that?
If we can't use `non_empty` could you at least change `v` to `slug`? I like more expressive variable names, if possible.

Reply:
A dependency for a single function seemed like an overkill. That's why I removed it. I have correctly named all parameters.

Reply:
I'm ok with the names.

My reasons for that dependency - something like `Empty` trait seems to be missing among standard Rust traits ; I don't like to write `!` ; I want to make it a little bit less verbose. But these are personal opinions, your solution is probably better for (at least) beginners.